packages/@n8n/benchmark/.eslintrc.js

```diff
@@ -0,0 +1,45 @@
+const sharedOptions = require('@n8n_io/eslint-config/shared');
+
+/**
+ * @type {import('@types/eslint').ESLint.ConfigData}
+ */
+module.exports = {
+	extends: ['@n8n_io/eslint-config/node'],
+
+	...sharedOptions(__dirname),
+
+	parserOptions: {
+		project: './tsconfig.json',
+	},
+
+	ignorePatterns: ['scenarios/**'],
+
+	rules: {
+		'n8n-local-rules/no-plain-errors': 'off',
+		'n8n-local-rules/no-uncaught-json-parse': 'off',
+		'n8n-local-rules/no-dynamic-import-template': 'error',
+		'n8n-local-rules/misplaced-n8n-typeorm-import': 'error',
+		'n8n-local-rules/no-type-unsafe-event-emitter': 'error',
+		complexity: 'error',
+
+		'import/order': 'off',
+		'import/extensions': 'warn',
+		'@typescript-eslint/ban-ts-comment': ['warn', { 'ts-ignore': true }],
+		'@typescript-eslint/no-explicit-any': 'warn',
+		'@typescript-eslint/no-base-to-string': 'warn',
+		'@typescript-eslint/prefer-nullish-coalescing': 'warn',
+		'@typescript-eslint/no-redundant-type-constituents': 'warn',
+		'@typescript-eslint/ban-types': 'warn',
+		'@typescript-eslint/no-unsafe-enum-comparison': 'warn',
+		'@typescript-eslint/no-unsafe-declaration-merging': 'warn',
+	},
```

Comment:
Can we please make sure all deviations from the parent config are needed and make sense? e.g.

- `no-uncaught-json-parse` is useful
-`no-dynamic-typeorm-import` does not apply to this package as it has no typeorm
- `@typescript-eslint/ban-ts-comment` is listed as `// TODO: Remove this` in other packages so we should start off this new package by disallowing it outright
- `no-type-unsafe-event-emit` is only for cli
- etc.

Reply:
Good catch! Removed (almost) all the disabled ones.